jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt

```diff
@@ -64,7 +58,8 @@ class SqsWindow(private val project: Project) : CoroutineScope by ApplicationThr
             AwsIcons.Resources.Sqs.SQS_TOOL_WINDOW
         )
 
-        fun getInstance(project: Project) = ServiceManager.getService(project, SqsWindow::class.java)
+        fun getInstance(project: Project): SqsWindow? = ServiceManager.getService(project, SqsWindow::class.java)
```

Comment:
`service()` will shorten this and remove the null-able

Reply:
changed to `project.service` already